diff --git a/dist/esm/start-manifest-plugin/plugin.js b/dist/esm/start-manifest-plugin/plugin.js
index 4a536eb6fb3a2b7f9e61a6205fa16e17530f3673..b8a0f95fedd5151417abf5ad2c3dff9e2e833865 100644
--- a/dist/esm/start-manifest-plugin/plugin.js
+++ b/dist/esm/start-manifest-plugin/plugin.js
@@ -70,9 +70,7 @@ function startManifestPlugin(opts) {
               chunksByFileName.set(bundleEntry.fileName, bundleEntry);
               if (bundleEntry.isEntry) {
                 if (entryFile) {
-                  throw new Error(
-                    `multiple entries detected: ${entryFile.fileName} ${bundleEntry.fileName}`
-                  );
+                  console.warn(`multiple entries detected: ${entryFile.fileName} ${bundleEntry.fileName}`)
                 }
                 entryFile = bundleEntry;
               }
